### PR TITLE
Added the sriov_ib project

### PR DESCRIPTION
**Pull request by abdallahyas**
Added the IPoIB CNI CI project to the scripts.

### CODE REVIEW

**Comment by abdallahyas** [comment]
@moshe010

File: sriov_ib_cni_install.sh [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+
+cd $WORKSPACE
+
+echo "Get CPU architechture"
+export ARCH="amd"
+if [[ $(uname -a) == *"ppc"* ]]; then
+   export ARCH="ppc"
+fi
+
+function configure_multus {
```
Comment (moshe010) [inline sriov_ib_cni_install.sh]: is this duplicate to the common code?

**Review by moshe010 — CHANGES_REQUESTED** [review]
make sure you use the common code

**Comment by abdallahyas** [comment]
@moshe010 added the code to set the exclusive mode to the install script.

File: sriov_ib_cni_stop.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+for ds in $(kubectl -n kube-system get ds |grep kube|awk '{print $1}'); do
+    kubectl -n kube-system delete ds $ds
+done
+
+for sc in $(screen -ls|grep multus|awk '{print $1}'); do
+    screen -X -S $sc quit
+done
+
+kill $(ps -ef |grep local-up-cluster.sh|grep $WORKSPACE|awk '{print $2}')
```
Comment (moshe010) [inline sriov_ib_cni_stop.sh]: why this is not in command methods

**Review by moshe010 — CHANGES_REQUESTED** [review]

**Review by moshe010 — APPROVED** [review]

**Pull request merged**